contrib/localnet/orchestrator/smoketest/cmd/smoketest/local/utils.go

```diff
@@ -249,11 +236,13 @@ func waitKeygenHeight(
 	// wait for keygen to be completed. ~ height 30
 	keygenHeight := int64(60)
 	logger.Print("⏳ wait height %v for keygen to be completed", keygenHeight)
-	for {
-		time.Sleep(2 * time.Second)
+	for i := 0; ; i++ {
+		time.Sleep(1 * time.Second)
```

Comment:
Maybe we can just wait for the height to increase rather than use a fixed sleep time?